platform-sdk/swirlds-platform-core/src/testFixtures/java/com/swirlds/platform/test/fixtures/event/RandomEventUtils.java

```diff
@@ -87,14 +93,21 @@ public static BaseEventHashedData randomEventHashedDataWithTimestamp(
                         otherParent.getGeneration(),
                         otherParent.getBaseEvent().getHashedData().getBirthRound());
 
+        final List<OneOf<PayloadOneOfType>> hapiTransactions = new ArrayList<>();
```

Comment:
we should not call them HAPI transactions. HAPI is a hedera concept, the platform should be independent of it